Makefile.conf

```diff
@@ -35,9 +35,9 @@ COQMF_CMDLINE_COQLIBS =
 ###############################################################################
 
 COQMF_LOCAL=0
-COQMF_COQLIB=/home/au543200/.opam/4.07.1/lib/coq/
-COQMF_DOCDIR=/home/au543200/.opam/4.07.1/doc/
-COQMF_OCAMLFIND=/home/au543200/.opam/4.07.1/bin/ocamlfind
+COQMF_COQLIB=/home/kasserne/.opam/4.11.1/lib/coq/
```

Comment:
This file probably should not be checked in, or at least the paths should be regenerated